### PR TITLE
ci: split integrations test to speed up

### PR DESCRIPTION
**Pull request by lhy1024**
### What problem does this PR solve?

Issue Number: Ref #4399

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by bufferflies — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -215,8 +215,11 @@ basic-test: install-tools
 
 ci-test-job: install-tools dashboard-ui
 	@$(FAILPOINT_ENABLE)
-	CGO_ENABLED=1 go test -timeout=15m -tags deadlock -race -covermode=atomic -coverprofile=covprofile -coverpkg=./... $(shell ./scripts/ci-subtask.sh $(JOB_COUNT) $(JOB_INDEX))
-	@ for mod in $(shell ./scripts/ci-subtask.sh $(JOB_COUNT) $(JOB_INDEX) 1); do cd $$mod && $(MAKE) ci-test-job && cd $(ROOT_PATH) > /dev/null && cat $$mod/covprofile >> covprofile; done	
+	if [[ $(JOB_INDEX) -le 10 ]]; then \
+	CGO_ENABLED=1 go test -timeout=15m -tags deadlock -race -covermode=atomic -coverprofile=covprofile -coverpkg=./... $(shell ./scripts/ci-subtask.sh $(JOB_COUNT) $(JOB_INDEX)); \
+	else \
+	for mod in $(shell ./scripts/ci-subtask.sh $(JOB_COUNT) $(JOB_INDEX)); do cd $$mod && $(MAKE) ci-test-job && cd $(ROOT_PATH) > /dev/null && cat $$mod/covprofile >> covprofile; done; \
```
Comment (HuSharp) [inline Makefile:221]: Maybe can extract `$(shell ./scripts/ci-subtask.sh $(JOB_COUNT) $(JOB_INDEX))` out of loop

**Review by JmPotato — COMMENTED** [review]
You may need to update the repo configs to make sure the newly added chunks be set as required like: https://github.com/ti-community-infra/configs/pull/546

**Comment by lhy1024** [comment]
> You may need to update the repo configs to make sure the newly added chunks be set as required like: [ti-community-infra/configs#546](https://github.com/ti-community-infra/configs/pull/546)

ok, I will update it later.

**Review by disksing — APPROVED** [review]

**Review by binshi-bing — APPROVED** [review]

**Pull request merged**